includes/widgets/testOne.php

```diff
@@ -0,0 +1,289 @@
+<?php
```

Comment:
@Mchris-hub Change the file name to `table.php` and update the class name.

Reply:
I changed the file name and I updated the class name

Reply:
Can you please push your changes? I am still seeing the old file name.